### DISCUSSION TITLE
CLI upgrade with pacman

### DISCUSSION THREAD

**Discussion by ksz16** [rern/rAudio#1247] [General]
First of all, thank you for your work - rAudio is really great software. In fact, I've only recently started my adventure with it and for now I'm in the testing phase.

At the very beginning I encountered a problem: after upgrading via CLI (pacman -Syy && pacman -Syu) I do a reboot. After this operation, the RPi starts up, but I have no communication with it - I can't open its home page in the browser. I also can't connect to the RPi via ssh - the message shows: "ssh: connect to host 192.168.x.x port 22: No route to host".

I've tried several times and each time ended up having to flash the image from scratch. Am I doing something wrong? Is rAudio not allowed to CLI upgrade with pacman like other Arch Linux derivatives? I noticed while updating the repository list that the rAudio repo is listed. So I thought it was possible to upgrade the rAudio addon to a new version using pacman. Was my assumption wrong? 

I would greatly appreciate clarification and any suggestions, especially since I already know that rAudio will stay with me as my only player/MPD server.

I am using RPi2 (BCM2836) and rAudio-RPi2-20230212.img.xz

**Comment by R101G** [comment]
@ksz16  If you are not using a wifi dongle, this sounds like the interface renaming problem. Edit the SD card with a PC and either add _net.ifnames=0_  to the /boot/cmdline.txt file, or put a .network file in /etc/systemd/network which matches your interface name.

The problem might have been fixed by a later rAudio update.

Edit: After re-reading your post, I think you might believe that updating the kernel also updates rAudio. It doesn't.

**Comment by ksz16** [comment]
@R101G  Thank you very much for such a quick response and clarification.
In my previous post I forgot to describe the details of my internet connection. I do not use a wifi dongle - I connect via ethernet. In addition, my RPi2 has DHCP reservation on the router and always gets the same IP address on my local network. These settings work great. The problem only occurs after executing the pacman -Syu command. 
It would be nice if this problem would be fixed in the next versions. And now at least I know a workaround for it.
Thank you very much once again.

**Comment by R101G** [comment]
@ksz16 Try updating rAudio via the addon before you use pacman, as it may already be fixed. The problem is due to a change in the Arch Linux OS. If the DHCP changes, you can _ssh root@raudio.local_ from a Linux machine.

If you add to the cmdline.txt file, remember everything has to be on that one line.

**Comment by ksz16** [comment]
@R101G I did as you wrote. First, I updated rAudio via GUI. Then I updated the system with pacman and restarted the RPi. After rebooting I was not able to connect either using my address from DHCP or using _raudio.local_ (both via browser and via SSH). When I added _net.ifnames=0_ at the end of the cmdline.txt file, communication was restored and I was able to log in via SSH and open the home page in the browser (using both: my address from DHCP and _raudio.local_)

**Comment by R101G** [comment]
Over to you @rern :)
